dumper/lib-ext-hive-metastore/src/main/java/com/google/edwmigration/dumper/ext/hive/metastore/HiveMetastoreThriftClient_Superset.java

```diff
@@ -498,4 +500,28 @@ public void close() throws IOException {
       throw new IOException("Unable to shutdown Thrift client.", e);
     }
   }
+
+  @Override
+  public ImmutableList<String> getCatalogsAsJsonl() throws TException {
+    TSerializer serializer = new TSerializer(new TSimpleJSONProtocol.Factory());
+    com.google.edwmigration.dumper.ext.hive.metastore.thrift.api.superset.GetCatalogsResponse
+        catalogs = client.get_catalogs();
```

Comment:
I recommend adding an import and dropping the package prefix here as well as in lines 521 and 524.

Reply:
Done.